src/sql/workbench/contrib/dashboard/browser/core/dashboardPage.component.html

```diff
@@ -8,28 +8,30 @@
 	[actions]="panelActions">
 	<div #toolbar [style.display]="showToolbar ? 'block': 'none'" class="editor-toolbar">
 	</div>
-	<tab [visibilityType]="'visibility'" *ngFor="let tab of tabs" [title]="tab.title" class="fullsize"
-		[identifier]="tab.id" [canClose]="tab.canClose" [actions]="tab.actions" [type]="tab.type" [iconClass]="tab.iconClass">
-		<ng-template>
-			<dashboard-home-container *ngIf="tab.id === 'homeTab'; else not_home" [properties]="propertiesWidget"
-				[tab]="tab">
-			</dashboard-home-container>
-			<ng-template #not_home>
-				<dashboard-webview-container *ngIf="getContentType(tab) === 'webview-container'" [tab]="tab">
-				</dashboard-webview-container>
-				<dashboard-widget-container *ngIf="getContentType(tab) === 'widgets-container'" [tab]="tab">
-				</dashboard-widget-container>
-				<dashboard-modelview-container *ngIf="getContentType(tab) === 'modelview-container'" [tab]="tab">
-				</dashboard-modelview-container>
-				<dashboard-controlhost-container *ngIf="getContentType(tab) === 'controlhost-container'" [tab]="tab">
-				</dashboard-controlhost-container>
-				<dashboard-nav-section *ngIf="getContentType(tab) === 'nav-section'" [tab]="tab">
-				</dashboard-nav-section>
-				<dashboard-grid-container *ngIf="getContentType(tab) === 'grid-container'" [tab]="tab">
-				</dashboard-grid-container>
-				<dashboard-error-container *ngIf="getContentType(tab) === 'error-container'" [tab]="tab">
-				</dashboard-error-container>
+	<div [style.height]="showToolbar ? 'calc(100% - 37px)' : '100%'">
```

Comment:
Definitely put a comment here explaining why 37px (and probably include a comment next to the toolbar height that this should be updated if that ever changes)

That being said - were you not able to figure out how to get this to resize appropriately? This can't be the only way to achieve this and I'm really not a fan of this kind of hard coded stuff...

Reply:
no more hard coding, i am getting the runtime height now.